chapters/lang/expressions.tex

```diff
@@ -193,17 +195,63 @@ \subsection{Term comparison operators}
 
 Characters are compared using their byte representation. Strings and regular expressions are compared character by character. For \lstinline{===} and \lstinline{!==} operators comparisons between characters, strings and regular expressions always fail, otherwise characters and regular expressions are always converted to strings before \lstinline{==} and \lstinline{!=} comparison.
 
-TODO(Ania): which term is returned?
+No matter what comparsion was made, its result consist term copied from the second arggument.
 
 \subsection{Result operators}
 
-TODO(Ania): not, and, or
+Result operators are used to made logical expressions, such as conjunction, alternative, xor and negation. Their actions are analogous to those from which they took their name in classical mathematics.
```

Comment:
`xor` is called **exclusive disjunction**: https://en.wikipedia.org/wiki/Exclusive_or

Reply:
> Their actions are analogous to those from which they took their name in classical mathematics

> Their actions are analogous to their classical logic counterparts.